src/store/rocket.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash'
+const state = () => {
+    return {employees: null}
+  };
+  
+  const mutations = {
+    setEmployees(state, employees) {
+        state.employees = employees;
+      }
+  };
+  
+  const actions = {
+    async getEmployees ({ commit }) {
+        try{
+            const employees = await this.$http.$get("https://cdn.contentful.com/spaces/"+this.$config.spacesId+"/entries?access_token="+this.$config.contentfulAccessToken+"&content_type=team")
+            const itemsFields = ['fields.name', 'fields.position', 'fields.profilePic.sys.id','sys.id']
+            const itemsFilteredEmployees = _.map(employees.items, e => _.pick(e, itemsFields))
+            const includesFields = ['fields.file.fileName', 'fields.file.url', 'sys.id']
+            const includesFilteredEmployees = _.map(employees.includes.Asset, e => _.pick(e, includesFields))
+            const mergedFilteredEmployees = _.values(_.merge(_.keyBy(includesFilteredEmployees,'sys.id'), _.keyBy(itemsFilteredEmployees,'fields.profilePic.sys.id' ) ))
+            const formattedFinalFilteredEmployees = mergedFilteredEmployees.map(function(item){
+                return {id:item.sys.id, 
+                        name:item.fields.name, 
+                        position:item.fields.position, 
+                        imageId:item.fields.profilePic.sys.id,
+                        imageName:item.fields.file.fileName, 
+                        imageUrl:"http:"+ (item.fields.file.url)
```

Comment:
hmm interesting... for some reason i though I had to spend http: to the URL in order for the img tag to read the URL properly. but I removed it and it still works, learned something new!

Reply:
With absolute URLs where you don't know what the protocol is going to be, you can lead it with two slashes. this can be especially helpful when testing locally which is often just http and testing on staging/production which is often https.

```
//images.google.com
```